test/contrib/automotive/xcp/xcp.uts

```diff
@@ -0,0 +1,823 @@
+% Regression tests for the XCP
+~ vcan_socket needs_root linux
```

Comment:
I think it's still too much to exclude the whole file for non-root and non-llinux